src/SprykerSdk/Zed/ApplicationRoute/Business/Model/Parser/YvesResourceCreatorParser.php

```
@@ -48,13 +52,21 @@ public function getResourceCreatorList(string $resourceCreatorSearchPattern)
             $resourceCreator = $this->getResourceCreatorInfo($item->getPathname());
 
             try {
-                include_once $item->getPathname();
+                if (!is_readable($item->getPathname())) {
```

Should that throw an exception instead of silently doing nothing?